dns.go

```diff
@@ -75,21 +80,33 @@ func (api *API) DNSRecords(zoneID string, rr DNSRecord) ([]DNSRecord, error) {
 	if rr.Content != "" {
 		v.Set("content", rr.Content)
 	}
+
 	var query string
-	if len(v) > 0 {
+	var records []DNSRecord
+	page := 1
+
+	// Loop over makeRequest until what we've fetched all records
+	for {
+		v.Set("page", strconv.Itoa(page))
 		query = "?" + v.Encode()
-	}
-	uri := "/zones/" + zoneID + "/dns_records" + query
-	res, err := api.makeRequest("GET", uri, nil)
-	if err != nil {
-		return []DNSRecord{}, errors.Wrap(err, errMakeRequestError)
-	}
-	var r DNSListResponse
-	err = json.Unmarshal(res, &r)
-	if err != nil {
-		return []DNSRecord{}, errors.Wrap(err, errUnmarshalError)
-	}
-	return r.Result, nil
+		uri := "/zones/" + zoneID + "/dns_records" + query
+		res, err := api.makeRequest("GET", uri, nil)
+		if err != nil {
+			return []DNSRecord{}, errors.Wrap(err, errMakeRequestError)
+		}
+		var r DNSListResponse
+		err = json.Unmarshal(res, &r)
+		if err != nil {
+			return []DNSRecord{}, errors.Wrap(err, errUnmarshalError)
+		}
+		records = append(records, r.Result...)
+		if len(records) >= r.ResultInfo.Total {
```

Comment:
Thanks, I didn't see that field had been added. I'll add this to `ResultInfo` and use that instead. Much nicer.